test/test_ops.py

```diff
@@ -165,6 +169,20 @@ def unsupported(dtype):
 
         self.assertEqual(supported_backward_dtypes, claimed_backward_supported, msg=msg)
 
+    # Tests that the function and its (array-accepting) reference produce the same
```

Comment:
"array" -> "ndarray" to clarify that it's a NumPy array that the references accept

Reply:
Fixed in the recent commit, thanks!